_data/hipaa-apps.yml

```diff
@@ -0,0 +1,57 @@
+features:
+  - title: Service discovery
+    description: Apps must look up service endpoints via environment variables, or DNS, or specific IP addresses, or a client-library.
+    available: "yes"
+
+  - title: Service mesh
+    description: After apps successfully find a service, they also need to authenticate to it, encrypt the communication channel, balance the load, and route traffic in specific ways.
+    available: "yes"
```

Comment:
We don't have service mesh yet.

Reply:
Changed to "Coming soon" for continuity with the index page's table in c165964.